src/splitInteger.test.js

```diff
@@ -4,18 +4,38 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+  const value = 12;
```

Comment:
make your life easier 

test(`should split a number into equal parts
  if a value is divisible by a numberOfParts`, () => {
  expect(splitInteger(8, 4))
    .toEqual([2, 2, 2, 2]);
});